docs/hld/projects/publish/high-level-design.md

```diff
@@ -0,0 +1,22 @@
+# High Level Design
+* According to the [design](https://www.figma.com/file/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?node-id=4684%3A7130) the published project can have its own:
+    * name
+    * description
+    * tags
+    * creators
+    * scenario thumbnail
+* User can see frequency maps, but not features ([refs](https://vizzuality.atlassian.net/browse/MARXAN-864?focusedCommentId=13137))
+* We shouldn’t need to show protected areas for public projects in public view (non-authenticated users) ([refs](https://vizzuality.atlassian.net/browse/MARXAN-863?focusedCommentId=13138))
+
+# To be declared 
+
+Things that still need some spike/analysis/decision. As they may not be 
+straightforward, we should consider writing down all pros/cons of all 
+possible solutions that team can come up with, finally making a decision and 
+describing why given solution was picked (and others were discarded).
+
+* [Architecture](./../../../adr/0006_published_projects.md)
+* Handling scenario thumbnail
+    * since users should be allowed to choose which scenario of the project to use for thumbnails, if the scenario chosen (as "highlight", let's say) is deleted, we may either want to prevent deletion (I'd say this is not desirable) or ask the user to pick another scenario to highlight before deleting it, or fall back to no map
+    * if users want to share a project that doesn't have any scenarios yet, we may want to prevent the project from being shared (it wouldn't make much sense at this stage, though I can see users may want to "get done with it" and set it as public even before they start working on it)
```

Comment:
I don't see any reason not to but this may be stakeholder decision at this point. Shouldn't planning area / planning units be enough?